test/tests/actions/get_actions/tests

```diff
@@ -3,5 +3,6 @@
     type = RunApp
     input = test_get_actions.i
     expect_out = 'Getting actions is successful'
+    pbs = false
   [../]
 []
```

Comment:
I realize I probably should have included this (and the rest of these) changes in a separate PR. These are the tests that simply fail, with no hope of fixing. Okay we can fix them, but it will require duplicate files in our git repo.

The issue is how our input files are listing where other files necessary for the test to complete are located:
```pre
[Mesh]
  file = ../meta_action/square.e
  uniform_refine = 4
[]

[TestGetActions]
[]
```

The file copy routines do not parse input files and so has no clue about this square.e file located in a relative directory (which is wrong when PBS creates a new sub-directory and runs from there).